src/ReaderTaskEither.ts

```diff
@@ -1,11 +1,16 @@
 import * as E from 'fp-ts/Either'
-import { Lazy, pipe } from 'fp-ts/function'
+import { identity, Lazy, pipe } from 'fp-ts/function'
 import * as IO from 'fp-ts/IO'
 import * as IOE from 'fp-ts/IOEither'
 import * as O from 'fp-ts/Option'
+import * as R from 'fp-ts/Reader'
+import * as RE from 'fp-ts/ReaderEither'
+import * as RT from 'fp-ts/ReaderTask'
 import * as RTE from 'fp-ts/ReaderTaskEither'
 import * as T from 'fp-ts/Task'
 import * as TE from 'fp-ts/TaskEither'
+import { ValueOf } from '.'
+import * as $H from './Has'
 import * as $S from './struct'
 
 export const pick =
```

Comment:
It could be a good chance to test `pics` too, even if is tested indirectly.

Reply:
All right!